test/integration_test.tsx

```diff
@@ -111,30 +111,14 @@ function addStaticTests(render: (el: ReactElement) => Wrapper) {
       );
     });
   }
-
-  if ((render as any) !== renderToString) {
-    it('returns contents of fragments', () => {
```

Comment:
I copied this test into each renderer's `describe` block since this new option changes the behavior and I wanted to reuse these tests for when the option is turned on.